pull_request_template.md

```diff
@@ -2,11 +2,18 @@
 <!-- Add a description of the fix or feature here -->
 
 ### Checklist
-- [ ] The version field in `package.json` of the package you have made changes in is incremented following [semantic versioning](https://semver.org/) and using alpha release tagging
-- [ ] The box that allows repo maintainers to update this PR is checked
-- [ ] I tested locally to make sure this feature/fix works
-- [ ] I have run `yarn file-check`, `yarn type-check` & `yarn build` to confirm there are not any associated errors
-- [ ] This PR passes the Circle CI checks
+- [ ] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
+- [ ] Check the box that allows repo maintainers to update this PR
+- [ ] Test locally to make sure this feature/fix works
+- [ ] Run `yarn file-check`, `yarn type-check` & `yarn build` to confirm there are not any associated errors
```

Comment:
```suggestion
- [ ] Run `yarn check-all` to confirm there are not any associated errors
```
Added a command that runs all 3 a few weeks back

Reply:
yessssss love the streamlining!